### ISSUE TITLE
Different score results when running multiple times.

### ISSUE THREAD

**Issue by dkuku** [tugascript/keyword-extraction-rs#14]
Hi @tugascript. 
I'm currently learning how to interface elixir with rust and I created a wrapper around your library.
Few things I spotted:
There are different function names in rake and text_rank: `word` vs `keyword`, `phares` vs `phrases` (that's a typo) - It would be good to unify the interface.
Running the same code multiple times returns different results. I made a test module but every time I'm tests the score is slightly different:
```elixir
-      assert_value ExKeywordExtractionNative.tr_get_word_score(ref, "rust") == 5.044811248779297                                                                                 
+      assert_value ExKeywordExtractionNative.tr_get_word_score(ref, "rust") == 5.044812202453613  
```
If you wan't to check the tests you need to clone
https://github.com/dkuku/ex_keyword_extraction
and after installing elixir run this inside the folder:
```
mix deps.get
mix test
```

**Comment by tugascript** [comment]
Ah all the algorithms are unsuprevised, so exact scores are not guaranteed on each run. I will fix the typo tomorrow.

As for unifying the interfaces, I'll do that on version 2 so there is no breaking change now. I just need to add parallel support for YAKE, and I'm waiting for a PR that reduces the `String` usage of the library.

**Comment by tugascript** [comment]
I will fix the interface on the release 2, the typo is fixed on version 1.4.3
